package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "build": "ember build",
     "start": "ember server",
-    "test": "ember try:each"
+    "test": "npm run test-lib && npm run test-ember",
+    "test-ember": "ember try:each",
+    "test-lib": "tape tests-node/**/*-test.js"
```

Comment:
I haven't used tape before, mainly mocha.  But I'm just this is fine.

As for tests-node, that's exactly where I've put them in the past.